### DISCUSSION TITLE
[Question] 如何自建系统集成？通过api调用触发执行实例，但是参数无法传递过来怎么办？How to integrate with third-party platforms? The execution instance is triggered by api call, but the parameters cannot be passed, what should I do?

### DISCUSSION THREAD

**Discussion by luojieio** [apache/dolphinscheduler#6354] [Ideas]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

如何自建系统集成？通过api调用触发执行实例，但是参数无法传递过来怎么办？How to integrate with third-party platforms? The execution instance is triggered by api call, but the parameters cannot be passed, what should I do?
比如：我想实例监控告警的时候触发，告警会传递一些告警信息，比如业务，项目，阈值，通知人，这些信息如何传递到工作流？
For example: I want to trigger when an instance monitoring alarm is triggered. The alarm will transmit some alarm information, such as business, project, threshold, and informant. How can this information be transmitted to the workflow?
### DolphinScheduler Version

1.3.6

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by crazycarry** [comment]
自己构建那个json模板，然后通过变量生成运行的json，目前那个json 是定死的，不能动态扩展

**Comment by DylanMao1996** [comment]
我们最近在做海豚集成当前业务，走自动化部署，目前海豚参数data返回是null。需要修改service拿到返回值。来实现自动化。
Recently we are doing dolphin integration current business, go automatic deployment, currently dolphin parameter data return is null. You need to modify the service to get the return value. To automate.

**Comment by Aaronx-Gallagher** [comment]
你好，怎么才能使用api接口创建可以执行的工作流实例呢，我使用了api创建了工作流但是无法启动，并且该工作流中的任务定义中也没有我定义的任务。
